components/src/components/molecules/PageButtons/PageButtons.tsx

```diff
@@ -83,16 +94,24 @@ export const PageButtons = ({
 
   if (total > max) {
     if (alwaysShowFirst && start > 1) {
-      array[0] = -1
-      array.unshift(1)
-    } else if (start > 1) {
+      if (showElipsis) {
+        array[0] = -1
```

Comment:
Let's change this to a more descriptive number type, such as a named enum, or a string